source/docs/articles/drupal/major-version-drupal-upgrades.md

```diff
@@ -6,26 +6,42 @@ category:
   - managing
 keywords: drupal, upgrade, upgrading, revision upgrade, update, updating
 ---
-The best practice for doing a major Drupal revision upgrade (e.g. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
+The best practice for doing a major Drupal revision upgrade (i.e. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
 
 Also, Pantheon needs to track the proper upstream Git history for your site to deliver core updates. By starting a new site for the upgrade, you ensure that future core updates will be available via the Dashboard.
 
-## Set up the Code and Migrate the Data
+The best way to think about upgrading to the next major revision of Drupal on Pantheon is the same as when you get a new laptop or smartphone: first you set up both devices side-by-side, then you migrate the data from the old to the new.
```

Comment:
Clarify, "set up both devices side-by-side," - doe it mean, "Install all of the applications on the new machine,"?